### DISCUSSION TITLE
Pictures inserted as inline image after v0.61

### DISCUSSION THREAD

**Discussion by gingerkitti** [TriliumNext/Trilium#4513] [General]
Hi there. Is there still a convenient way to insert image inline, especially for image included through drag&drop and clip&paste?

I remembered that previously there was as option for change the picture to inline image, but now it seems missing. 

Of course I can still do this by making the image inserted standalone note, but this workaround lacks convenience and it makes the image notes show in every parent note, since trilium has removed the function "hide images included in a note" after  v0.61(#546)

<sup>This discussion maybe both suits Ideas and Q&A , so I put it in General</sup>

**Comment by zadam** [comment]
> I remembered that previously there was as option for change the picture to inline image, but now it seems missing.

I don't think there was such an option.

The default way to insert images is since 0.61 to make them an attachment of a note - if I may ask, in what way is this a problem for you? It behaves quite similarly as if the image was inline ...

**Reply by FordringWang** [reply]
Dear zadam,
Copy images to export notes is very difficult, I have an example my friend asked me for a note, and this note has a lot of pictures. I want to quickly copy and paste to the word document, I found that images could not be copied, I just copy the images one by one I think this is a big job...I haven't thought any good way for this yet. do you have any suggestions for this scenario?

**Reply by meichthys** [reply]
My recommendation would be to "share" the note (Note > Basic Properties > Shared Toggle) and send the shared note link with your friend. The Trilium shared links are not easily guessable, so it should be relatively private. 

Note: This does assume that your trilium instance is accessible from the internet.

**Reply by FordringWang** [reply]
Thanks for your reply, this is a good idea. It would be even greater if future Trilium Notes could support quick copying of images, I really like this note by the way.
